nextflow.config

```diff
@@ -0,0 +1,59 @@
+params {
+    // Required Parameters
+    outdir            = "/vast/scratch/users/$USER/results"
+    inputdir          = "$projectDir/example"
+}
+
+// Milton Slurm Configuration
+process {
+    executor = 'slurm'
+    cache    = 'lenient'
+}
+executor {
+    queueSize         = 100
+    queueStatInterval = '10 sec'
+    pollInterval      = '10 sec'
+    submitRateLimit   = '10 sec'
+}
+
+//Tower Configuration
+/* tower {
+    enabled = true
+    workspaceId = '103725969693338'  //Change this to your workspace id
+    endpoint = 'https://tower.services.biocommons.org.au/api'
+} */
+
+
+profiles {
+    debug {
+        dumpHashes             = true
+        process.beforeScript   = 'echo $HOSTNAME'
+        cleanup = false
+    }
+    milton{
+        conda.enabled          = true
+        cleanup                = true
+        apptainer.enabled      = true
+        apptainer.autoMounts   = true
+        singularity.enabled    = false
+        process {
+            withLabel:Test {
+                module = 'python/3.9.17' 
+                // Add ':' then another module name, to load mulitple modules
+            }    
+           
+        }
+    }
+    conda {
+        conda.enabled          = true
+        process {
+            withLabel:Test {
+                conda ="$projectDir/envs/test.yaml"
+                // Must have $projectDir to work on seqera platform
+            }
+        }
+    }
+}
+
+// Load modules.config for DSL2 module specific options
+includeConfig 'config/modules.config'
```

Comment:
What is the advantage of splitting the config into two different files here?

Reply:
We cannot set process specifics in profile and also general process{} in same nextflow.config 
https://www.nextflow.io/docs/latest/config.html#config-profiles